crawlerr/models/crawl_models.py

```
@@ -0,0 +1,110 @@
+"""
+Data models for web crawling operations.
+"""
+from typing import Optional, List, Dict, Any, Union
+from datetime import datetime
+from enum import Enum
+from pydantic import BaseModel, Field, HttpUrl, field_validator, ConfigDict
```

Ruff (0.12.2)

`pydantic.HttpUrl` imported but unused

Remove unused import: `pydantic.HttpUrl`

(F401)